test/performance/README.md

```diff
@@ -111,13 +111,88 @@ To generate the sample workload for KubeStellar performance experiments proceed
     - 10 config maps containing a 2048-character random string
 
 
-
-
 5. Clean up the generated workload Kubernetes API objects from your cluster:
 
    ```bash
    $ cd $CL2_DIR
    $ ./cleanup.sh
    ```
 
-### Collect the creation & update timestamps for workload benchmark (OPTIONAL) (TO DO: WORK-IN-PROGRESS)
\ No newline at end of file
+
+### Collect the creation & update timestamps for workload benchmark (OPTIONAL)
+
+
+To collect the creation and status_update timestamps for the benchmark workload objects proceed as following:  
+
+1. Run the metrics collection script:
+
+   a) cd into the `test/performance` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash
+   $ cd .$HOME/kubestellar/test/performance
+   ```
+
+   b) Run the metrics collection script:
+
+   ```bash 
+   $ python3 metrics_collector.py <kubeconfig> <wds-context-name> <its-context-name> <wec-context-name> <number-of-namespaces> <output-directory> 
+   ```
+
+   For example:
+   
+   ```bash 
+   $ python3 metrics_collector.py $HOME/.kube/config wds1 its1 cluster1 2 $HOME/data
```

Comment:
I tested this. The script ran without producing any error messages, and it also produced no output files. Was I supposed to run it before the cleanup script? If so then there is a problem with the narrative order in these instructions.

Reply:
I tested again without running the cleanup script. Still no output file. BTW, should I look for them in `.` or `<output-directory>` ? If the latter, then the example should be adjusted. Even so, I found no output in either place.

```shell
(.venv) bash-3.2$ ls $HOME/data
(.venv) bash-3.2$ python3 metrics_collector.py $HOME/.kube/config wds1 its1 cluster1 2 $HOME/data
(.venv) bash-3.2$ find $HOME/data
/Users/mspreitz/data
(.venv) bash-3.2$ ls
README.md		cleanup.sh		metrics_collector.py	setup-clusterloader2.sh
__pycache__		kshelper.py		requirements.txt	workloads
(.venv) bash-3.2$ kubectl get ns
NAME                 STATUS   AGE
kube-system          Active   83m
kube-public          Active   83m
kube-node-lease      Active   83m
default              Active   83m
kubestellar-report   Active   82m
nginx                Active   81m
perf-exp-1           Active   2m16s
perf-exp-0           Active   2m16s
(.venv) bash-3.2$ 
(.venv) bash-3.2$ find . -name "perf-exp*"
(.venv) bash-3.2$ 
```

Reply:
@MikeSpreitzer, I just pushed a patch for that, could you re-run again the following command?

```
$ python3 metrics_collector.py $HOME/.kube/config wds1 its1 cluster1 2 $HOME/data
```

Yes, you should run it without running the cleanup script. I will fix the README to address that now.